server/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const app = express();
+const port = 3003;
+var db = require ('../database/index.js');
+const bodyParser = require('body-parser');
+
+app.use(express.static('./public'));
+app.use(bodyParser.urlencoded({extended: false}))
+app.use(bodyParser.json());
+
+app.get('/:experience_id', (req, res) => {
```

Comment:
you may want to change this route.  this is the URL that Robin is going to use to callup the site.  it looks like he's going to get all your photos instead.